src/frontend/apps/desk/src/features/teams/components/TeamInfo.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import IconGroup from '@/assets/icons/icon-group2.svg';
+import { Box, Card, Text } from '@/components';
+import { useCunninghamTheme } from '@/cunningham';
+
+import { TeamResponse } from '../api/types';
+
+interface TeamInfoProps {
+  team: TeamResponse;
+}
+
+export const TeamInfo = ({ team }: TeamInfoProps) => {
+  const { t } = useTranslation();
+  const { colorsTokens } = useCunninghamTheme();
+
+  return (
+    <Card className="m-b" style={{ paddingBottom: 0 }}>
+      <Box className="m-b" $direction="row" $align="center" $gap="1.5rem">
+        <IconGroup
+          width={44}
+          color={colorsTokens()['primary-text']}
+          aria-label={t('icon group')}
+          style={{
+            flexShrink: 0,
+          }}
+        />
+        <Box>
+          <Text as="h3" $weight="bold" $size="1.25rem" className="mt-0">
+            {t('Members of “{{teamName}}“', {
+              teamName: team.name,
+            })}
+          </Text>
+          <Text>
+            {t('Add people to the “{{teamName}}“ group.', {
+              teamName: team.name,
+            })}
+          </Text>
```

Comment:
nit: font-size is of 13px on the design file !

Reply:
[64aa8ad](https://github.com/numerique-gouv/people/pull/49/commits/64aa8adebdfb1b797a4b66af063a5a1c026ddf92)